src/components/Legend/examples/basic.jsx

```diff
@@ -113,7 +113,7 @@ export default createClass({
 				<br />
 				<br />
 
-				<ToolTip isExpanded={true} direction='right' alignment='end'>
+				<ToolTip isExpanded={true} direction='right' alignment='end' isLight={true}>
```

Comment:
nice, the current example looks a little ridiculous.  Thanks for fixing :) 

You'll need to accept these changes in Chromatic.  There is a link in for it in the Travis build summary.

Reply:
Found the link. Accepted the changes.